src/widget/wtrackmenu.cpp

```diff
@@ -138,6 +139,166 @@ void WTrackMenu::createMenus() {
         //: Reset metadata in right click track context menu in library
         m_pClearMetadataMenu->setTitle(tr("Reset"));
     }
+
+    if (featureIsEnabled(Feature::SearchRelated)) {
+        // TODO: Create a new subclass of QMenu?
```

Comment:
Yes please. This is IMO too much feature-specific code to cram into createMenus.

Reply:
The TODO comment still needs to be removed.